virtualfish/environment.fish

```diff
@@ -0,0 +1,56 @@
+function __vfext_environment_activate --on-event virtualenv_did_activate
+    if test -f $VIRTUAL_ENV/virtualfish-environment
+        for line in (cat $VIRTUAL_ENV/virtualfish-environment)
```

Comment:
I don't think there's anything wrong with getting the file contents via `cat`, but Fish actually has a [`read` command built-in](https://fishshell.com/docs/current/commands.html#read). Here's the method I used:
```
function __vfext_environment_activate --on-event virtualenv_did_activate
    if test -r $env_file_path
        while read line;
            [ ... logic that sets the environment variables ... ]
        ; end < $env_file_path
    end
end
```
Again, I haven't done any performance profiling to compare how the two might differ in speed, but at least wanted to mention this method in case you decide it has merit.